### PR TITLE
Remove deprecated fields for assignment and queue RPC

### PR DESCRIPTION
**Pull request by terencechain**
Remove deprecated fields for assignment and queue RPC endpoints for v3

### CODE REVIEW

File: validator/client/attest.go [hunk 1]
```diff
@@ -186,7 +186,7 @@ func (v *validator) duty(pubKey [fieldparams.BLSPubkeyLength]byte) (*ethpb.Dutie
 		return nil, errors.New("no duties for validators")
 	}
 
-	for _, duty := range v.duties.Duties {
+	for _, duty := range v.duties.CurrentEpochDuties {
```
Comment (terencechain) [inline validator/client/attest.go:189]: Double check. `Duties` should equal to `CurrentEpochDuties`

File: validator/client/validator.go [hunk 2]
```diff
@@ -692,7 +692,7 @@ func (v *validator) subscribeToSubnets(ctx context.Context, res *ethpb.DutiesRes
 // validator assignments are unknown. Otherwise returns a valid ValidatorRole map.
 func (v *validator) RolesAt(ctx context.Context, slot types.Slot) (map[[fieldparams.BLSPubkeyLength]byte][]iface.ValidatorRole, error) {
 	rolesAt := make(map[[fieldparams.BLSPubkeyLength]byte][]iface.ValidatorRole)
-	for validator, duty := range v.duties.Duties {
+	for validator, duty := range v.duties.CurrentEpochDuties {
```
Comment (terencechain) [inline validator/client/validator.go:695]: Same as above

**Comment by james-prysm** [comment]
This PR also contains breaking changes for the UI

File: proto/prysm/v1alpha1/validator.proto [hunk 3]
```diff
@@ -471,11 +471,9 @@ message DutiesRequest {
 }
 
 message DutiesResponse {
-    repeated Duty duties = 1 [deprecated = true];
+    repeated Duty current_epoch_duties = 1;
 
-    repeated Duty current_epoch_duties = 2;
-
-    repeated Duty next_epoch_duties = 3;
+    repeated Duty next_epoch_duties = 2;
```
Comment (prestonvanloon) [inline proto/prysm/v1alpha1/validator.proto:476]: Do not change field numbers!!

See my suggestion in your other PR https://github.com/prysmaticlabs/prysm/pull/11248#discussion_r948328529

File: proto/prysm/v1alpha1/beacon_chain.proto [hunk 4]
```diff
@@ -720,21 +720,13 @@ message ValidatorQueue {
...
-    repeated bytes activation_public_keys = 2 [(ethereum.eth.ext.ssz_size) = "?,48", deprecated = true];
-
-    // Ordered list of public keys awaiting exit. 0th index is the next key to
-    // be processed.
-    repeated bytes exit_public_keys = 3 [(ethereum.eth.ext.ssz_size) = "?,48", deprecated = true];
-
     // Ordered list of validator indices awaiting activation. 0th item in the list is the
     // next validator index to be processed.
-    repeated uint64 activation_validator_indices = 4 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v3/consensus-types/primitives.ValidatorIndex"];
+    repeated uint64 activation_validator_indices = 2 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v3/consensus-types/primitives.ValidatorIndex"];
```
Comment (prestonvanloon) [inline proto/prysm/v1alpha1/beacon_chain.proto:725]: Do not change field numbers!!

See my suggestion in your other PR https://github.com/prysmaticlabs/prysm/pull/11248#discussion_r948328529

**Review by prestonvanloon — CHANGES_REQUESTED** [review]

**Comment by terencechain** [comment]
As it turns out, these are still needed for the UI. We don't have time to deprecate these before V3 release. Since it's not a priority, I recommend that we open an issue and aim this for V4

**Pull request closed without merging**